compute/rn/release-information/release-notes-22-12.adoc

```diff
@@ -667,6 +667,7 @@ A new  "Compliance ID" column is added to indicate the compliance-related issues
 
 
 === Backward Compatibility for New Features
+NOTE: For Prisma Cloud Enterprise (SaaS) tenants, any defenders that are older than 22.01 will no longer be able to connect to the console. 
```

Comment:
Thanks for this @jjeanclaude . These RN are for on-prem.

@divyabhushan, please see if we can add this to the PCEE RN for Compute.

Reply:
included in [PR-891](https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/891)